src/main/java/seedu/bookmark/model/book/comparators/BookPagesReadComparator.java

```diff
@@ -0,0 +1,31 @@
+package seedu.bookmark.model.book.comparators;
+
+import java.util.Comparator;
+
+import seedu.bookmark.model.book.Book;
+
+/**
+ * Sorts {@code Book}s based on {@code PagesRead} in ascending order.
```

Comment:
Maybe this could be modified to be percentage of book read? As different books may have varying pages, e.g. a 500 page book vs a 100 page book and might be more useful to sort based on completion rate. Just an idea.

Reply:
I was deciding between pages read and percentage of book read, and I felt that pages read was more meaningful, but we can discuss this in our next meeting!